### PR TITLE
Reorganize examples

### PR DESCRIPTION
**Pull request by disktnk**
Removed `test_rt.py`. Instead, add ResNet50 example on example directory.

- [x] add another example, YOLOv2?
- [x] run example script on CI

### CODE REVIEW

File: examples/resnet50/export.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
+import argparse
```
Comment (shinh) [inline examples/resnet50/export.py]: Could you add an example command line?

Comment (disktnk) [reply examples/resnet50/export.py]: @shinh add quick docs on comments.

I added YOLOv2Tiny example. Could you check it? thx.

Comment (shinh) [reply examples/resnet50/export.py]: Thanks! If there is no particular reason, I like usage at the beginning of code. Is it common to put the example usage after import lines (perhaps it's specified in pep8)? If it's the case, the current location is fine to me.

Comment (disktnk) [reply examples/resnet50/export.py]: PEP8 allow top comments before import if the comments follow pydoc style, so I fixed, Thnaks!

**Pull request merged**